### PR TITLE
adding container to quiz game

### PR DESCRIPTION
**Pull request by ibrahimaljarad**
this is just a container t quiz game with the test file,

### CODE REVIEW

File: src/containers/QuizGamePage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import QuizGame from "../../components/QuizGame";
+import Navbar from "../../components/Navbar/index";
+
+const QuizGamePage = () => {
+  return (
+    <>
+      <Navbar />
```
Comment (tmansuroglu) [inline src/containers/QuizGamePage/index.jsx]: Navbar will be visible without Navbar component here.

Comment (tmansuroglu) [reply src/containers/QuizGamePage/index.jsx]: I will add a fix for it

**Review by tmansuroglu — APPROVED** [review]
Nice work!

File: src/containers/QuizGamePage/index.jsx (hunk 1 above)
Comment (ibrahimaljarad) [reply src/containers/QuizGamePage/index.jsx]: PTAL

File: src/locales/en.json [hunk 2]
```diff
@@ -93,6 +93,12 @@
     }
   },
   "games": {
+    "quizgame":{
+      "screenshot":{
+        "title":"Quiz Game",
+        "src":"https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7"
+      }
+    },
```
Comment (tmansuroglu) [inline src/locales/en.json]: "quizgame":{
      "title":"Quiz Game",
      "screenshot": "https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7",
      ...
    }

I was referring to something like this

Comment (ibrahimaljarad) [reply src/locales/en.json]: PTAL

Comment (tmansuroglu) [reply src/locales/en.json]: nice

File: src/locales/en.json [hunk 3]
```diff
@@ -96,6 +96,10 @@
     "title": "Games"
   },
   "games": {
+    "quizgame": {
+      "title": "Quiz Game",
```
Comment (majdajroudi) [inline src/locales/en.json]: I think we should also be adding these to the ar.json and tr.json files

Comment (ibrahimaljarad) [reply src/locales/en.json]: PTAL

File: src/App.jsx [hunk 4]
```diff
@@ -25,6 +26,7 @@ function App() {
         <Route path="/login" component={""} />
         <Route path="/games/memorygame" component={MemoryGame} />
         <Route exact path="/games/puzzle" component={PuzzlePage} />
+        <Route path="/games/quizgame" component={QuizGamePage} />
```
Comment (louisrli) [inline src/App.jsx:29]: @tmansuroglu can you please rename your component to MemoryGamePage. you can see that there is an inconsistency here in the naming

File: src/locales/ar.json [hunk 5]
```diff
@@ -96,6 +96,11 @@
     "title": "ألعاب"
   },
   "games": {
+    "quizgame": {
+      "title": "لعبة الاسئلة",
+      "screenshot": "https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7"
```
Comment (louisrli) [inline src/locales/ar.json]: this doesn't have to be in the json. unless its different per translation

just declare a variable `const QUIZ_GAME_IMAGE = <the url>` and use it in code

File: src/locales/en.json [hunk 6]
```diff
@@ -96,6 +96,10 @@
...
+      "screenshot": "https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7"
```
Comment (louisrli) [inline src/locales/en.json]: notice how the 3 are duplicated = already a warning

File: src/locales/ar.json (hunk 5 above)
Comment (louisrli) [reply src/locales/ar.json]: also you added these translations but where are they used?

Comment (ibrahimaljarad) [reply src/locales/ar.json]: actually i added this image and title so tarkan can use them in the game page, we will have a game page has the 3 games with this screenshot of each game.

File: src/App.jsx (hunk 4 above)
Comment (tmansuroglu) [reply src/App.jsx:29]: sure

File: src/locales/ar.json (hunk 5 above)
Comment (ibrahimaljarad) [reply src/locales/ar.json]: removed the screenshot

File: src/locales/en.json (hunk 6 above)
Comment (ibrahimaljarad) [reply src/locales/en.json]: object removed

**Review by louisrli — APPROVED** [review]
so you removed the translations completely? pr looks fine as it is though

**Comment by ibrahimaljarad** [comment]
> so you removed the translations completely? pr looks fine as it is though

i just removed the screenshot object which we added it as extra for the games page

**Comment by ibrahimaljarad** [comment]
@Ammar-64  can you merge please

**Comment by louisrli** [comment]
this PR used to have a translations file and now it doesnt

**Comment by louisrli** [comment]
that translations file also had a translated title

**Review by louisrli — CHANGES_REQUESTED** [review]
my understanding was you were gonna delete only the line that had the screenshot

**Comment by ibrahimaljarad** [comment]
> my understanding was you were gonna delete only the line that had the screenshot

the previous object with the translation has nothing to do with this pr, this pr is only for making a container for the quiz game so when tarkan asked for only a title and a screenshot for adding it to the general games page i added it here,so it has nothing to do with the translation

**Review by louisrli — APPROVED** [review]

**Comment by louisrli** [comment]
and that's how it should be, a clean separation of PRs in that case :)

**Review by majdajroudi — APPROVED** [review]

**Comment by Ammar-64** [comment]
@ibrahimaljarad the branch has a conflict, can you fix it please?

**Comment by ibrahimaljarad** [comment]
@Ammar-64  can we merge now please,

**Pull request merged**